source/Calamari/Deployment/SpecialVariables.cs

```diff
@@ -106,6 +112,8 @@ public static class Features
         public static class Action
         {
             public const string Name = "Octopus.Action.Name";
+            public const string SkipRemainingConventions = "Octopus.Action.SkipRemainingConventions";
```

Comment:
Fair enough.  My comment only applied if there was *zero* reason anything outside of Calamari needed to know about the variable.